app-text/master-pdf-editor/master-pdf-editor-5.3.20.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit desktop xdg-utils
+
+DESCRIPTION="A complete solution for viewing and editing PDF files"
+HOMEPAGE="https://code-industry.net/free-pdf-editor/"
+SRC_URI="https://code-industry.net/public/${P}_qt5.amd64.tar.gz"
+
+LICENSE="master-pdf-editor"
+SLOT="0"
+KEYWORDS="~amd64"
+RESTRICT="mirror"
+
+RDEPEND="
+	>=dev-qt/qtcore-5.4.1:5
```

Comment:
What you should you instead is make sure the package depends on Qt version 5. Change these lines to:
```
dev-qt/qtcore:5=
```
You should also change the lines below related to Qt. Make sure to read https://devmanual.gentoo.org/general-concepts/dependencies/ if you want to find out more about slots and subslots.

Reply:
No. Qt uses subslots for a different purpose.

Reply:
So, should I make the change or not?
The only reason that I have made it as such is that these are the versions specified in the software (same for the sane-backends).

Reply:
Please just disregard the subslot comments.